syft/workers/base.py

```diff
@@ -371,6 +378,7 @@ def execute_command(self, message: tuple) -> "pointers.PointerTensor":
             if type(_self) == str and _self == "self":
                 _self = self
             if sy.torch.is_inplace_method(command_name):
+                # TODO[jvmancuso]: figure out a good way to generalize the above check
```

Comment:
Please create an issue for this and link here

Reply:
https://github.com/OpenMined/PySyft/issues/2530